### ISSUE TITLE
message editing

### ISSUE THREAD

**Issue by gsbhasin123** [Nheko-Reborn/nheko#149]

**Comment by deepbluev7** [comment]
This is a duplicate of #134 . Please check the bug tracker before opening new issues, especially since github already suggests duplicates and please provide a longer description instead of just a title, when opening an issue! Thank you!

**Comment by gsbhasin123** [comment]
Ah, sorry. Thank you!
I didn't provide a description since it was self explanatory, also, GitHub didn't suggest any duplicates for me, although it usually does (strange?)

I'll be waiting on editing support then :D this is a feature I really require

The up arrow should also be able to be used for editing, currently it duplicates the message instead of updating it.
